src/MongoDB.Driver.Core/Core/Configuration/ConnectionString.cs

```diff
@@ -457,11 +457,11 @@ public ConnectionString Resolve()
             var host = GetHostNameForDns();
 
             var client = new LookupClient();
-            var response = client.Query(srvPrefix + host, QueryType.SRV);
+            var response = client.Query(srvPrefix + host, QueryType.SRV, QueryClass.IN);
```

Comment:
This change was made for symmetry with the QueryAsync method (QueryClass.IN is the default value for the new third parameter).